### DISCUSSION TITLE
Why use tartufo over gitleaks?

### DISCUSSION THREAD

**Discussion by bittner** [godaddy/tartufo#513] [Q&A]
I found out about tartufo recently, and I tried - unsuccessfully - to understand the differences compared to [gitleaks](https://github.com/gitleaks/gitleaks). There doesn't seem to be much content on this on the Web.

Could someone explain the key differences, maybe also with more tools, other than gitleaks, if any, and what makes tartufu particularly stand out?

**Comment by sushantmimani** [comment]
Hi @bittner ,

While both these tools provide similar functionalities, Tartufo uses a combination of entropy checks and regex matching by default to find secrets. If I understand correctly, gitleaks uses only regex matching by default. Tartufo also allows remote scanning on repos.
